tee-worker/cli/src/trusted_base_cli/commands/litentry/set_scheduled_mrenclave.rs

```diff
@@ -32,23 +32,26 @@ use teerex_primitives::MrEnclave;
 
 #[derive(Parser)]
 pub struct SetScheduledMrenclaveCommand {
+	/// AccountId in ss58check format
+	account: String,
+	/// Sidechain block number from which mrenclave is in effect
 	bn: SidechainBlockNumber,
-	//hex encoded
+	/// Hex encoded mrenclave
 	mrenclave: String,
 }
 
 impl SetScheduledMrenclaveCommand {
 	pub(crate) fn run(&self, cli: &Cli, trusted_cli: &TrustedCli) {
-		let root = get_pair_from_str(trusted_cli, "//Alice");
+		let who = get_pair_from_str(trusted_cli, &self.account);
```

Comment:
This will not work, unfortunately. Because you can't get the pair from public address except for well known keys like `//Alice``